contrib/packs/actions/pack_mgmt/setup_virtualenv.py

```diff
@@ -98,9 +98,15 @@ def _setup_pack_virtualenv(self, pack_name):
                           (pack_name, virtualenv_path))
 
     def _create_virtualenv(self, virtualenv_path):
-        self.logger.debug('Creating virtualenv in "%s"' % (virtualenv_path))
```

Comment:
Well it basically tells me that this feature is not suited as a pack but should most likely be rolled up into a first-class platform feature. If we get strict on the sandboxing of runners things could break very easily if the config is not passed into the python runner process etc.